sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MyApplication.java

```diff
@@ -24,6 +24,7 @@ public void onCreate() {
     //      });
     //       */
     //    });
+    //throw new RuntimeException("Startup Crash!");
```

Comment:
Did you mean to add a comment explaining how to trigger a startup crash here or should this code be removed?

Reply:
yeah I was just testing, we could remove it